server/src/main/java/com/aws/greengrass/cli/CLIEventStreamAgent.java

```diff
@@ -686,9 +686,14 @@ public GetLocalDeploymentStatusResponse handleRequest(GetLocalDeploymentStatusRe
                         Topics deploymentStatusDetailsTopics =
                                 deployment.findTopics(DEPLOYMENT_STATUS_DETAILS_KEY_NAME);
                         DeploymentStatusDetails deploymentStatusDetails = new DeploymentStatusDetails();
-                        DetailedDeploymentStatus detailedDeploymentStatus = detailedDeploymentStatusFromString(
-                                Coerce.toString(deploymentStatusDetailsTopics.find(DEPLOYMENT_DETAILED_STATUS_KEY)));
-                        deploymentStatusDetails.setDetailedDeploymentStatus(detailedDeploymentStatus);
+                        if (deploymentStatusDetailsTopics.find(DEPLOYMENT_DETAILED_STATUS_KEY) != null) {
```

Comment:
don't bother checking for null here, it is covered by detailedDeploymentStatusFromString

Reply:
Makes sense, will remove this check